plugins/modules/nfs_exports_info.py

```diff
@@ -0,0 +1,133 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright: (c) 2025, Samaneh Yousefnezhad <s-yousefenzhad@um.ac.ir>
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: nfs_exports_info
+
+short_description: Extract folders, IPs, and options from /etc/exports
+
+description: >
+  This module retrieves and processes the contents of the /etc/exports file from a remote server,
+  mapping folders to their corresponding IP addresses and access options.
+
+author:
+  - Samaneh Yousefnezhad (@yousefenzhad)
+
+options:
+  output_format:
+    description:
+      - The format of the returned mapping.
+      - If set to C(ips_per_share), output maps shared folders to IPs and options.
+      - If set to C(shares_per_ip), output maps IPs to shared folders and options.
+    required: true
+    type: str
+    choices: ['ips_per_share', 'shares_per_ip']
+"""
+
+EXAMPLES = r"""
+- name: Get IPs and options per shared folder
+  fava.infra.nfs_exports_info:
+    output_format: 'ips_per_share'
```

Comment:
I would add a `register` to make it more obvious that you should register the result:
```suggestion
    output_format: 'ips_per_share'
  register: result
```